plugins/outputs/orionres/processor/resultdef.go

```diff
@@ -14,13 +14,15 @@ type ResultDefDim struct {
 }
 
 type ResultDef struct {
-	Name              string
-	MetricServiceName string
-	Enabled           bool
-	RemapResFacts     bool
-	Dims              []*ResultDefDim
-	DimAttribs        map[string]string
-	ResFacts          map[string]*info.ResSetFact
+	Name                string
+	MetricServiceName   string
+	Enabled             bool
+	RemapResFacts       bool
+	TestDim             bool
+	PrimaryDimensionSet string
+	Dims                []*ResultDefDim
+	DimAttribs          map[string]string
+	ResFacts            map[string]*info.ResSetFact
 }
 
 func newResultDef(defName string, setDefs *info.MetricDefs) *ResultDef {
```

Comment:
below, if `defName` is not found, is it okay if `PrimaryDimensionSet` is empty string?  I assume that `TestDim` should be false so no need to explicitly set it as such.

Reply:
Good point.  I think I will default to TestDim=true, and PrimaryDimensionSet="test".  This allows setting the configured test if set and doesn't seem to cause a problem if test filed left empty.